MobaLiveCD.txt

```diff
@@ -0,0 +1,21 @@
+[Section]
+ Name = MobaLiveCD 
+Version = 2.1
+License = GPL
+
+ Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu".
+ 
+Size = 1.52MB
+ 
+Category = 12
+URLSite = https://www.mobatek.net/labs_mobalivecd.html
+URLDownload = https://www.mobatek.net/exes/MobaLiveCD_v2.1.exe
+CDPath = none
+ 
+ 
```

Comment:
Some suspicious empty lines.

Reply:
Sorry for the problems. Feel free to correct the wrong things you find.